spec/models/book_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe Book, type: :model do
+  subject(:book) { described_class.new(gender: gender, author: author, image: image, title: title, editor: editor, year: year) }
+
+  let(:gender) { Faker::Book.genre }
```

Comment:
This is the first time I notice this. Our bad, should've told you earlier that it should be genre istead of gender.